src/compiler/mcp.md

```diff
@@ -1,190 +0,0 @@
-# Major Change Proposals
```

Comment:
Where did the content of this file go? In `proposals-and-stabilizations.md`?

I think this file served ~two~ three very important purposes:
- A clear explaination of what "MCP" is and stands for
- Provide a clear link (good for SEO) to explain this tool
- Explain in detail the procedure

I should probably do a deeper A/B check - was all the semantic content transferred in the new file?

Reply:
I think all of the content was moved over - at least that which I thought was still relevant. I didn't think MCPs should be on their own page because they should be presented in the context of the approval processes that require it.